.changeset/nice-schools-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-adr': minor
```

Comment:
Hey @jhaals 👋 

I actually suggested this be a `minor` since this is technically a breaking change due to the constructor of the public `AdrClient` interface being changed

Reply:
Hey @kuangp shoot, you're right!